apps/aechannel/src/aesc_fsm.erl

```diff
@@ -3089,10 +3315,21 @@ init(#{opts := Opts0} = Arg) ->
                 _   -> StateInitF(Initiator)
             end,
     ClientMRef = erlang:monitor(process, Client),
+    BlockHashDelta =
+        case maps:find(block_hash_delta, Opts) of
+            error ->
+                #bh_delta{not_newer_than = 0, %% backwards compatibility
+                          not_older_than = 10};
```

Comment:
What is the reasoning for these values?

Reply:
Fallback default values

Reply:
I meant why 10 and not 100 ?

Reply:
While 100 is great for fast-mining tests, 0-10 generations is expected to be "from 30 minutes in the past till now". 100 would mean last 5 hours which might be a bit too long